server/renderer.js

```diff
@@ -27,11 +27,11 @@ function render(req, res, data, context) {
 
     }
 
-    if (req.query.hasOwnProperty('bemjson')) return res.send('<pre>' + JSON.stringify(bemjson, null, 4) + '</pre>');
+    if (req && req.query.hasOwnProperty('bemjson')) return res.send('<pre>' + JSON.stringify(bemjson, null, 4) + '</pre>');
 
     const html = BEMHTML.apply(bemjson);
 
-    return res.send(html);
+    return !send ? res.send(html) : html;
```

Comment:
@skad0 I think `send` is a bad name of this mod, but I cann't think a better.
This modifier is needed to manage `return`, we give pure html or we do `res.send(html)`